kotlin-native/Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -385,8 +385,8 @@ private fun processCLib(flavor: KotlinPlatform, cinteropArguments: CInteropArgum
         KotlinPlatform.NATIVE -> {
             val outLib = File(nativeLibsDir, "$libName.bc")
             val compilerCmd = arrayOf(compiler, *compilerArgs,
-                    "-emit-llvm", "-c", outCFile.absolutePath, "-o", outLib.absolutePath)
-            runCmd(compilerCmd, verbose)
+                    "-emit-llvm", "-x", "c", "-c", "-", "-o", outLib.absolutePath)
+            runCmd(compilerCmd, verbose, redirectInputFile = File(outCFile.absolutePath))
```

Comment:
A comment explaining why we redirect the file from stdin instead of passing it directly would be useful.

Reply:
Sure, tried to capture it above.